### PR TITLE
Convert to PWA

### PR DESCRIPTION
**Pull request by stephanritscher**
Without this PR, Firefox for Android offers an "add to home screen" button which is basically only a link which is opened in the regular Firefox app.
With this PR, Firefox for Android offers to "Install" the website as Progressive Web App on the home screen. This separates the website in a separate activity/process in Android and hides some of regular browser controls.

The requiremens for Firefox for Android to recognize a PWA are explained [here](https://webplatform.news/issues/2020-08-25).
Basically you need a secure site (thus I changed http references to https - not all of them would be necessary, but better safe than sorry), a webmanifest with an large (192x192) icon and an (short_)name.

### CODE REVIEW

**Comment by weatherstorm** [comment]
Thanks for adding this to the skin.
